pyproject.toml

```diff
@@ -1,21 +1,13 @@
 [build-system]
 build-backend = "setuptools.build_meta"
-requires = [
-  "setuptools>=61.2",
-]
+requires = [ "setuptools>=61.2" ]
```

Comment:
some unrelated to the patch. We can remove the pyproject formater from pre-commit and fix this file. It has been updated some time ago splitting some things (like ruff configs) unnecessary -- could be in another PR

Reply:
+1